packages/cyberstorm/src/components/Layout/LoginLayout/BetaLoginLayout.tsx

```diff
@@ -19,30 +25,42 @@ export function BetaLoginLayout() {
             Login to Thunderstore Beta
           </div>
           <div className={styles.loginButtons}>
-            <div className={styles.loginButton}>
-              <Button
-                colorScheme="discord"
-                label="Discord"
-                leftIcon={<FontAwesomeIcon icon={faDiscord} fixedWidth />}
-                paddingSize="large"
-              />
-            </div>
-            <div className={styles.loginButton}>
-              <Button
-                colorScheme="github"
-                label="GitHub"
-                leftIcon={<FontAwesomeIcon icon={faGithub} fixedWidth />}
-                paddingSize="large"
-              />
-            </div>
-            <div className={styles.loginButton}>
-              <Button
-                colorScheme="overwolf"
-                label="Overwolf"
-                leftIcon={<OverwolfLogo />}
-                paddingSize="large"
-              />
-            </div>
+            {props.discordAuthUrl ? (
+              <div className={styles.loginButton}>
+                <a href={props.discordAuthUrl}>
```

Comment:
You could add `onClick((e) => e.preventDefault))` to prevent the user from getting directed to external site. That would be in line with recent changes done by Vilppe.

Reply:
At first I got bit confused, but I'm guessing you meant that; it should be a disabled link instead of the button not being there?

Reply:
Oh, you probably meant at the storybook end, right.

Reply:
I am confuse. We want people to go and visit the link? If you are talking about the storybooks end; I'd have to make a new component for the "fakeLinks" and then have that have a preventDefault option as a prop, which is out of scope for now.

Reply:
Alright, my bad.

Using `/` as the url prop in the story would make the link to lead to Storybooks frontpage, which while not perfect, would IMO be an improvement. Couldn't find a clever way to make the user stay on the same page (due to use of iframes) 😞. I don't consider this a blocker though.

Reply:
Yeah, that makes sense, I'll change that up